daemon/logger/loginfo.go

```diff
@@ -127,3 +129,45 @@ func (info *Info) ImageFullID() string {
 func (info *Info) ImageName() string {
 	return info.ContainerImageName
 }
+
+// InfoMap returns information about a container as a map.
+func (info *Info) InfoMap(keyMod func(string) string) map[string]string {
```

Comment:
The idea was that existing logger implementations might already expose some of the information available using `InfoMap` but, for historical reasons, with different key names. The `keyMod` function passed to the `InfoMap` function can be used to convert the new InfoMap name to the old one to not break the naming for existing consumers. This is already used in the fluentd logger because the new names we agreed on above do not match the existing ones for the information that can already be obtained (container ID and name).

I have not checked if the existing names match between the different loggers (because the change to the other loggers would possibly happened in a different PR), so I don't know if you want to change the new names or change fluentd's names to the new scheme. Depends on if you want to keep compatibility or want to have consistency.